config/routes.rb

```diff
@@ -4,10 +4,11 @@
       collection do
         get :faq, path: 'casto-kladene-otazky'
         get :register_step1, path: 'registracia'
-        post :register_step2, path: 'registracia-statutar'
-        post :register_step3, path: 'registracia-nastavenie'
-        post :register_step4, path: 'registracia-zabezpecenie'
-        post :register_step5, path: 'registracia-odoslanie'
+        post :register_step2, path: 'registracia-osoba'
```

Comment:
asi skor subjekt (ako pravny subjekt)

```suggestion
        post :register_step2, path: 'registracia-subjekt'
```